models/cnn/AlexNet/test_AlexNet.py

```diff
@@ -29,7 +30,148 @@ def test_overfit(model):
 
 
 if __name__ == "__main__":
```

Comment:
Don't use the main function. We are using pytest so just decorate the functions with @pytest.xxx.xxx. pytest would recognize the files of pattern test_*.py and run them. In your local, run `pytest models/cnn/AlexNet/test_AlexNet.py`.